computer_vision/noise_augmentation.py

```diff
@@ -0,0 +1,99 @@
+import glob
+import os
+import random
+from string import ascii_lowercase, digits
+
+import cv2
+import numpy as np
+
+# Params
+IMAGE_DIR = "1"
+OUTPUT_DIR = "out"
+NOISE_TYPE = "salt-and-pepper"  # ("salt-and-pepper" or "gaussian")
+ALPHA = 0.1  # ("salt-and-pepper" amount or "gaussian" variance)
+
+
+def main() -> None:
+    """
+    Get images list from input dir.
+    Update new images.
+    Save images in output dir.
+    """
+    img_paths = get_dataset(IMAGE_DIR)
+    print("Processing...")
+    new_images, paths = update_images(img_paths, NOISE_TYPE, ALPHA)
+
+    for index, image in enumerate(new_images):
+        # Get random string code: '7b7ad245cdff75241935e4dd860f3bad'
+        letter_code = random_chars(32)
+        file_name = paths[index].split(os.sep)[-1].rsplit(".", 1)[0]
+        file_root = f"{OUTPUT_DIR}/{file_name}_NOISY_{letter_code}"
+        cv2.imwrite(f"{file_root}.jpg", image, [cv2.IMWRITE_JPEG_QUALITY, 100])
+        print(f"Success {index+1}/{len(new_images)} with {file_name}")
+
+
+def get_dataset(img_dir: str) -> list:
+    """
+    - img_dir <type: str>: Path to folder contain images
+    Return <type: list>: List of images path
+    """
+    img_paths = []
+    for img_file_path in glob.glob(os.path.join(img_dir, "*")):
+        img_paths.append(img_file_path)
+    return img_paths
+
+
+def update_images(img_list: list, noise_type: str, alpha: float) -> tuple[list, list]:
```

Comment:
```suggestion
def update_images(img_list: list, noise_type: Literal["salt-and-pepper", "gaussian"], alpha: float) -> tuple[list, list]:
```
and include a `from typing import Literal` at the top